code/datums/status_effects/debuffs.dm

```diff
@@ -178,6 +178,29 @@
 /datum/status_effect/pacifism/on_remove()
 	REMOVE_TRAIT(owner, TRAIT_PACIFISM, id)
 
+// used to track if hitting someone with a cult dagger/sword should stamina crit.
+/datum/status_effect/cult_stun_mark
+	id = "cult_stun"
+	duration = 10 SECONDS // the when they stand back up, it will be over
+	alert_type = null
+	var/mutable_appearance/overlay
+
+/datum/status_effect/cult_stun_mark/on_apply()
+	. = ..()
+	if(!. || !ishuman(owner))
+		return
```

Comment:
Currently the parent doesn't do anything so I would remove the `. = ..()` and `!.`